src/translations/dynamic-mapping-models-en.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+const dynamic_mapping_models_en = {
+    // --- models --- //
+    'models/GeneratorSynchronousThreeWindings':
```

Comment:
To be more consistent with the rest of the application, we should use dots instead of slashs for separators

Reply:
Yes! DONE